doc/source/data/user-guide.rst

```diff
@@ -17,6 +17,7 @@ show you how achieve several tasks.
     inspecting-data
     iterating-over-data
     saving-data
+    execution-configurations
```

Comment:
Wondering if we should move this lower since the page seems advanced. Wdyt?

Reply:
good point, moved the section after "working with X" sections. i didn't want to label the page explicitly as "advanced" since some of these options (verbose logging, max_errored_blocks) can be pretty useful for simple use cases like reading files from S3.